doc/specification.md

```diff
@@ -290,10 +297,10 @@ This leads to the following generated JSON schema:
   "description": "Interface properties",
   "type": "object",
   "properties": {
-    "$schema": {
-      "title": "Schema",
+    "formatVersion": {
+      "title": "ABAP File Format Version",
       "description": "Format version",
-      "type": "string"
+      "const": "1"
```

Comment:
I am not sure. Should we also add a type (like number or integer)? 
Values would then be changed to `1` instead of `"2"`.

Reply:
Change type to `number` is bad, because for JSON schema the following representation of integer `1` are equivalent

 `1` and `1.0` and `1.00` and others